contrib/kind.sh

```diff
@@ -24,7 +24,7 @@ run_kubectl() {
 # with Fedora32 Cloud, but it does not happen if we clean first the ovn-kubernetes resources.
 delete()
 {
-  kubectl --kubeconfig ${HOME}/admin.conf delete namespace ovn-kubernetes
+  timeout 5 kubectl --kubeconfig ${HOME}/admin.conf delete namespace ovn-kubernetes || true
   sleep 5
```

Comment:
is the sleep needed now?

Reply:
was just going to leave it to give some hardcoded buffer between tearing down docker containers and pods, cause of the Fedora crash that happens

Reply:
I guess I could try testing it, at the peril of crashing my own laptop :)

Reply:
nah forget it , I was curious because never used the timeout command
/lgtm